pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientBuilderImpl.java

```diff
@@ -155,6 +155,12 @@ public ClientBuilder maxNumberOfRejectedRequestPerConnection(int maxNumberOfReje
         return this;
     }
 
+    @Override
+    public ClientBuilder keepAliveIntervalSeconds(int keepAliveIntervalSeconds) {
```

Comment:
For consistency, in most other config options with times we're taking the interval and the time unit, then in the `ClientConfigurationData` we convert into a specific unit (which for this case seconds are perfectly fine).